aindfpa.f90

```diff
@@ -28,43 +29,43 @@ subroutine aindfpa
 
 !.......................................................................
 
-      ibox(1)="unset"
-      ibox(2)="unset"
-      ibox(3)="unset"
-      iuser="unset"
-      ioutput(1)=6
-      ioutput(2)=0
-      mnemonic="mnemonic"
+      setup0%ibox(1)="unset"
```

Comment:
gbw, i think we can/should move these to the conf mod. If we get all of them I can ditch the local pointer hack...